tabpy-server/setup.py

```diff
@@ -1,23 +1,27 @@
-import versioneer
 try:
     from setuptools import setup
 except ImportError as err:
     print("Missing Python module requirement: setuptools.")
     raise err
 
+def read_version():
+    with open('../VERSION') as h:
+        return h.read().strip()
+
+__version__=read_version()
```

Comment:
You might also want to have this in the package __init__ file as I guarantee external libraries may expect something like this to be available

Reply:
By library you mean Python package right? I interpret this repo as containing separate client and server packages, so I would think the most logical thing would be to have `__version__` defined in `__init__.py` for both of those packages.

I don't think it being in `tabpy.py` really does much as that's more of an executable module rather than an importable one, though certainly possible I'm misinterpreting intention here